### DISCUSSION TITLE
拟开发将日志推送到手机

### DISCUSSION THREAD

**Discussion by jischeng** [MaaAssistantArknights/MaaAssistantArknights#2749] [Ideas]
昨天刚使用MAA, 摸鱼电脑一般不连接屏幕, 现在有点想法, 将日志推送到手机.  

目前还没拟定方案, 摸鱼时打算拟定下方案, 现在初步设想是通过公众号/小程序(没深入了解, 初步构想)去请求当前已经生成的日志信息

构思时分两部, 一是如上请求日志信息, 二是通过公众号/小程序远程link start!处于开启状态的MAA(在远程电脑上开机状态下)

目前两步都没分析可行性, 忙完这阵会分析下试试.

如果有后续, 会在这条更新(可能没有后续, 三分钟热度就是我 )

**Comment by MessiahJK** [comment]
一 的话可以监听gui.log文件的变化（？
二 可以考虑使用类似于plus plus这种统一推送服务，让用户自己去配置token或webhock等方式

**Reply by jischeng** [reply]
感谢指点，论文水完就去搞

**Reply by bigqiao** [reply]
建议考虑用webhook的方式，通知属于周边生态，可以由第三方能力实现。
